src/test/java/org/folio/mr/service/UserCloningServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.mr.service;
+
+import feign.FeignException;
+import org.folio.mr.domain.dto.User;
+import org.folio.mr.service.impl.UserCloningServiceImpl;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class UserCloningServiceTest {
+
+  @Mock
+  private UserService userService;
+
+  @InjectMocks
+  private UserCloningServiceImpl userCloningService;
+
+  @Test
+  void testCloneWhenUserExists() {
+    // given
```

Comment:
Do we really need these comments in a 4-line test?

Reply:
removed